Mathlib/Data/BitVec/Lemmas.lean

```diff
@@ -1,199 +0,0 @@
-/-
-Copyright (c) 2020 Simon Hudon. All rights reserved.
-Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Simon Hudon, Harun Khan
--/
-import Mathlib.Data.Nat.Bitwise
-import Mathlib.Data.BitVec.Defs
-
-/-!
-# Basic Theorems About Bitvectors
-
-This file contains theorems about bitvectors.
--/
-
-namespace BitVec
-
-open Nat
-
-#noalign bitvec.bits_to_nat_to_list
-#noalign bitvec.to_nat_append
-
-variable {w v : Nat}
-
-theorem toFin_injective {n : Nat} : Function.Injective (toFin : BitVec n → _)
-  | ⟨_, _⟩, ⟨_, _⟩, rfl => rfl
-
-theorem toFin_inj {x y : BitVec w} : x.toFin = y.toFin ↔ x = y :=
-  toFin_injective.eq_iff
-
-theorem toNat_injective {n : Nat} : Function.Injective (BitVec.toNat : BitVec n → _) :=
-  Fin.val_injective.comp toFin_injective
```

Comment:
I think anything about `Function.Injective` should be kept, since we can't _state_ those lemmas any further upstream

Reply:
That should be stated downstream of Mathlib, not upstream.